src/managed/dir.proj

```diff
@@ -24,7 +24,7 @@
       <Output ItemName="buildCmdArgs" TaskParameter="BuildArgs" />
     </GetBuildArgsByFrameworks>
 
-    <Exec Command="$(DotnetToolCommand) build $(Args) %(buildCmdArgs.Identity)" 
+    <Exec Command="$(DotnetSdkToolCommand) build $(Args) %(buildCmdArgs.Identity)" 
```

Comment:
Yeah, 100% agreed there. This is the only purpose for the `GetBuildArgsByFrameworks` task, by the way. If these projects were hooked into the build regularly, we wouldn't need it.